pkg/cmd/cli/describe/printer.go

```diff
@@ -39,7 +39,7 @@ var (
 	templateColumns         = []string{"NAME", "DESCRIPTION", "PARAMETERS", "OBJECTS"}
 	policyColumns           = []string{"NAME", "ROLES", "LAST MODIFIED"}
 	policyBindingColumns    = []string{"NAME", "ROLE BINDINGS", "LAST MODIFIED"}
-	roleBindingColumns      = []string{"NAME", "ROLE", "USERS", "GROUPS"}
+	roleBindingColumns      = []string{"NAME", "ROLE", "USERS", "GROUPS", "SERVICE ACCOUNTS", "SUBJECTS"}
```

Comment:
> Probably should just be "SUBJECTS" with kind and name but we can do that later. Open a follow up issue.

I'm not sure I agree.  I think that users will still think in terms of users and groups, so presentations that help them do it are valuable.  "SUBJECTS" in this context only holds those subjects that I couldn't smash into users, groups, or service accounts.